prime-dotnet-webapi/Services/SiteService.cs

```diff
@@ -593,7 +593,7 @@ public async Task<IEnumerable<BusinessEvent>> GetSiteBusinessEventsAsync(int sit
         {
             return await _context.BusinessEvents
                 .Include(e => e.Admin)
-                .Where(e => e.SiteId == siteId && businessEventTypeCodes.Any(c => c == e.BusinessEventTypeCode))
+                .Where(e => e.SiteId == siteId && businessEventTypeCodes.Any(c => c == e.BusinessEventTypeCode) || e.Organization.Sites.Any(s => s.Id == siteId))
```

Comment:
Why was this change necessary?

Reply:
Business events related to Organizations don't show up otherwise

Reply:
Looking through the code, it doesn't appear that we show organization events anywhere in our app currently.
I'm not sure that showing all of the Organization evens on each site is the best place for it, but it's not really in scope for this ticket to make a screen for just Organization events. As it stands, this is probably the best place for it for now.

However, this method now behaves in two unexpected ways:
1) It returns both Site events and Organization events despite being called `GetSiteBusinessEventsAsync` and being on the `SiteService`. This is super minor, but maybe renaming it to `GetSiteRelatedBusinessEventsAsync` or something like that might be more clear? Makes the name longer though. Maybe add a short comment?

2) This service accepts a list of `BusinessEventType` codes to filter the return, but Organization Events wont be filtered out due to the `&&` operation having precedence over the `||` operation. This method currently returns events that 
 a) Match the site Id and are in the list of codes.
 or
 b) are org events on the site's parent org

to keep the filtering behavior, the line can be changed to :
```C#
.Where(e => businessEventTypeCodes.Any(c => c == e.BusinessEventTypeCode) && (e.SiteId == siteId || e.Organization.Sites.Any(s => s.Id == siteId))
```